src/defaults.js

```diff
@@ -47,19 +47,6 @@ module.exports = {
     thumbnailMaxWidth: 640,
     keepEncryptedProperties: false
   },
-  avatar: {
```

Comment:
Don't remove this block; we'll remove the src directory in one shot when the web client use using modules